content/already_bought_annuity.md

```diff
@@ -4,16 +4,20 @@ description: Find out about the new annuity rules set to change in April 2016.
 
 # If you’ve already bought an annuity
 
-Annuity rules will change in April 2016. You may be able to sell an existing annuity and use the money to get a lump sum or a [flexible income](/pension-pot-options#get-a-flexible-income).
+Annuity rules will change in April 2016. You may be able to sell an existing annuity and use the money to get a lump sum or a [flexible income](https://www.pensionwise.gov.uk/pension-pot-options#get-a-flexible-income).
```

Comment:
We should use relative links - `[flexible income](/pension-pot-options#get-a-flexible-income)` not `[flexible income](https://www.pensionwise.gov.uk/pension-pot-options#get-a-flexible-income)`

This is so that links work on our staging environment.